### PR TITLE
[FRONT-1740] Mobile overflow fixes

### PR DESCRIPTION
**Pull request by tumppi**
A couple of fixes that caused sponsorship and operators page to overflow on mobile.

1. Tips are forced to left shift on mobile
2. About dropdowns are now centered on mobile
3. Long sponsorship/operator names are no more overflowing

### CODE REVIEW

**Comment by tumppi** [comment]
@mondoreale Can you take over this PR. Feel free to make changes and merge when ready since I'll be on holiday.

File: src/components/ActionBars/AboutOperator.tsx
Comment (mondoreale) [inline src/components/ActionBars/AboutOperator.tsx:1]: Before

Screenshot 2023-12-15 at 4 19 23 PM


After

Screenshot 2023-12-15 at 4 18 16 PM

Comment (mondoreale) [reply src/components/ActionBars/AboutOperator.tsx:1]: Also, I'm not sure if the "stick to the center" is the visually expected way. Can we make it stick to the LHS of the toggle and only shrink when needed?

Screenshot 2023-12-15 at 4 22 45 PM

**Comment by mondoreale** [comment]
Ah right, I see the comment now. I take it over then, sure!

**Comment by mondoreale** [comment]
I replaced this PR with #1769.

**Pull request closed without merging**